src/player.rs

```diff
@@ -47,8 +48,6 @@ pub struct PlayerState {
     pub status: PlayStatus,
     pub position_ms: u32,
     pub position_measured_at: i64,
-    pub update_time: i64,
```

Comment:
It's tied to the changes i made in spirc, there is a new struct "State" which contains the whole system state (Player and Mixer) so I thought the update_time belongs to the State not the PlayerState